sources/Afup/Comptabilite/Facture.php

```diff
@@ -476,7 +478,9 @@ function genererFacture($reference, $chemin = null)
             utf8_decode($coordonnees['adresse']) . "\n" .
             utf8_decode($coordonnees['code_postal']) . "\n" .
             utf8_decode($coordonnees['ville']) . "\n" .
-            utf8_decode($pays->obtenirNom($coordonnees['id_pays'])));
+            utf8_decode($pays->obtenirNom($coordonnees['id_pays'])) .
+            $coordonnees['tva_intra'] ? ("\nTVA : ".utf8_decode($coordonnees['tva_intra'])) : null
```

Comment:
```suggestion
            ($coordonnees['tva_intra'] ? ("\nTVA : ".utf8_decode($coordonnees['tva_intra'])) : null)
```

![Capture d’écran du 2023-12-08 12-37-42](https://github.com/afup/web/assets/320372/cbc38a7b-291e-4c56-a61f-feb4fee1eef5)

mieux vaux ajouter les parenthèses, sans celles-ci les infos de société/service/adresse/cp/ville/pays disparaissent.

Reply:
Oops, bien vu ! Merci